template/partial/mythbusters/myth_3.html

```diff
@@ -0,0 +1,3 @@
+ASSUMPTION: The culture here is different, so Peace Corps staff will blame me for what happened.
+<div class="ui hr divider"></div>
```

Comment:
@AkankshaBodhankar No, the div should not close here because it is just for a divider line. Also there is no requirement of enclosing it in a tag because it is a partial html file which is being fed to another file. I hope you have gone through the complete flow, and you will clearly see what I have done.